.github/workflows/ci.yaml

```diff
@@ -4,6 +4,10 @@ on:
       - master
   pull_request:
 name: ci
+env:
+  CI: true
+  FORCE_COLOR: 2
```

Comment:
If this is specifically causing the tests to fail, let's maybe leave this out and come back around (the other changes are good)

Reply:
Commented it out for now, and yeah that was causing the test failures.

Reply:
I'm going to try with `shell: cmd` in another branch and see if this makes tests fail so that #33 is tested.

Reply:
My guess is that you'd need some kind of library to strip out the characters or adapt the test messages to match. The comment is there for future reference but it's a trivial thing anyway.